api/controllers/user.go

```diff
@@ -0,0 +1,127 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/mail"
+	"os"
+
+	"github.com/PedroChaparro/loomies-backend/interfaces"
+	"github.com/PedroChaparro/loomies-backend/models"
+	"github.com/gin-gonic/gin"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const uri = "mongodb://root:development@localhost:27017/"
+
+func Pruebas() {
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+
+	if err != nil {
+		fmt.Println("Mongo.connect() error: ", err)
+		os.Exit(1)
+	}
+	// Obtain the DB, by name. db will have the type
+	// *mongo.Database
+	db := client.Database("loomies")
+
+	// use a filter to only select capped collections
+	command := bson.D{{"create", "users"}}
```

Comment:
Is this to create the users collection? The collection is created automatically when the first insert occur.

Reply:
Same as previous case